addons/website_forum/models/forum_post.py

```diff
@@ -23,7 +24,7 @@ class Post(models.Model):
         'website.seo.metadata',
         'website.searchable.mixin',
     ]
-    _order = "is_correct DESC, vote_count DESC, write_date DESC"
+    _order = "is_correct DESC, vote_count DESC, bump_date DESC"
```

Comment:
Is this to support  the sort (and reverse) on the frontend?

Reply:
Actually no. I put it back to its original state.